app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -139,3 +135,45 @@
     expect($serviceMonitoringTarget->teams())->exists()->toBeTrue();
     expect($serviceMonitoringTarget->users())->exists()->toBeTrue();
 });
+
+test('it will validate multiple valid forms of URL and IP Address', function (array $validUrls, array $invalidUrls) {
+    asSuperAdmin();
+
+    foreach ($validUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasNoFormErrors();
+    }
+
+    foreach ($invalidUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasFormErrors(['domain']);
+    }
+})
+    ->with([
```

Comment:
Though it's fine to leave this how it is I would just note that this is not really a use case for datasets.

A dataset is used when there are multiple sets to run across a single test. This is a single set.